validator/validator-main.protoascii

```diff
@@ -4635,6 +4670,36 @@ tags: {
   }
 }
 
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "IMG"
+  spec_name: "amp-img > img (transformed)"
+  # Ideally we'd be able to use regular amp-img parent, but runtime currently
+  # requires an i-amphtml-ssr attribute on the amp-img. We could do away with
+  # that, though.
+  mandatory_parent: "amp-img (transformed)"
+  attrs: { name: "alt" }
```

Comment:
Do we need to require `i-amphtml-fill-content i-amphtml-replaced-content` which get added by the runtime?

Reply:
We could, but:

- This would require merging https://github.com/ampproject/amphtml/pull/29021 first (it fixes issues with specifying a mandatory class attr)
- It wouldn't allow the reverse order, which might be weird?

Reply:
> It wouldn't allow the reverse order, which might be weird?

Agree, not ideal. Could be OK in the case of transformed AMP.

Reply:
- Follow up next week after [cl/318548991](http://cl/318548991) (via #29021) lands here?
 - Would value_regex help implement an order-independent check?